ansible/roles/test/files/ptftests/py3/dhcpv6_counter_test.py

```diff
@@ -15,10 +15,13 @@
 DHCP6_Rebind = scapy.layers.dhcp6.DHCP6_Rebind
 DHCP6_Release = scapy.layers.dhcp6.DHCP6_Release
 DHCP6_Decline = scapy.layers.dhcp6.DHCP6_Decline
+DHCP6_Reconf= scapy.layers.dhcp6.DHCP6_Reconf
+DHCP6_InfoRequest = scapy.layers.dhcp6.DHCP6_InfoRequest
 DHCP6_Advertise = scapy.layers.dhcp6.DHCP6_Advertise
 DHCP6_Reply = scapy.layers.dhcp6.DHCP6_Reply
 DHCP6_RelayReply = scapy.layers.dhcp6.DHCP6_RelayReply
 DHCP6OptRelayMsg = scapy.layers.dhcp6.DHCP6OptRelayMsg
+DHCP6OptBootFileUrl = scapy.layers.dhcp6.DHCP6OptBootFileUrl # dhcp6opt out of client scope to test malformed counters
 
```

Comment:
@kellyyeh Your RP test failed because `ptf` command which is called by `test_bgp_speaker.py` loads all scripts under `ptftests` even scripts under `py3`. There is no DHCP6OptBootFileUrl for old version scapy, it only exists in `scapy 2.4.5`.
This is [introduction](https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/test/files/ptftests/py3/README.md).

_But for left Python2 scripts, they will still call ptf command, --test-dir is ptftests, it will load all scripts under ptftests **even scripts under subfolder py3**. So make sure it doesn't have incompatible issue when running ptf command._


```
>>> import scapy
>>> import ptf
>>> import ptf.testutils as testutils
>>> ptf.testutils
<module 'ptf.testutils' from '/usr/lib/python2.7/dist-packages/ptf/testutils.pyc'>
>>> IPv6 = scapy.layers.inet6.IPv6
>>> scapy.layers
<module 'scapy.layers' from '/usr/local/lib/python2.7/dist-packages/scapy/layers/__init__.pyc'>
>>> 
>>> DHCP6OptBootFileUrl = scapy.layers.dhcp6.DHCP6OptBootFileUrl
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'DHCP6OptBootFileUrl'
>>> 
```

I suggest you to add an if condition to avoid failure for python2, the following can resolve your problem.

```
if hasattr(scapy, "VERSION") and tuple(map(int, scapy.VERSION.split('.'))) >= (2, 4, 5):
    DHCP6OptBootFileUrl = scapy.layers.dhcp6.DHCP6OptBootFileUrl # dhcp6opt out of client scope to test malformed counters
```

Reply:
Thanks Zhaohui, I just changed the option to one that's compatible with both version